pkgs/development/ocaml-modules/ocaml-lsp/jsonrpc.nix

```diff
@@ -9,7 +9,9 @@
   lib,
   ocaml,
   version ?
-    if lib.versionAtLeast ocaml.version "5.02" then
+    if lib.versionAtLeast ocaml.version "5.2" then
+      "1.21.0"
+    else if lib.versionAtLeast ocaml.version "5.02" then
```

Comment:
"5.02" and "5.2" are equal versions. Only one test is necessary.

Reply:
:man_facepalming: